nautobot/core/settings.yaml

```diff
@@ -0,0 +1,1174 @@
+---
+"definitions":
+  "absolute_path":
+    "type": "string"
+  "callable":
+    "type": "string"
+  "regex":
+    "type": "string"
+  "relative_path":
+    "type": "string"
+"properties":
+  "ADMINS":
+    "default": []
+    "description": "Administrators' names and emails as a list of (name, email) tuples."
+    "details": |
+        For example:
+
+        ```python
+        ADMINS = [
+            ['Hank Hill', 'hhill@example.com'],
+            ['Dale Gribble', 'dgribble@example.com'],
+        ]
+        ```
+
+        Please see the [official Django documentation on `ADMINS`](https://docs.djangoproject.com/en/stable/ref/settings/#admins) for more information.
+    "items":
+      "items":
+        "type": "string"
+      "type": "array"
+    "type": "array"
+  "ALLOWED_HOSTS":
+    "default": []
+    "description": "A list of valid fully-qualified domain names (FQDNs) and/or IP addresses that can be used to reach the Nautobot service. (If provided as an environment variable, it should be a space-separated string, for example `NAUTOBOT_ALLOWED_HOSTS=\"localhost 127.0.0.1 example.com\"`)"
```

Comment:
Can we use the `>-` syntax to make these even easier to read (removing the need to escape double quotes here, breaking up long single lines)